docs/drill4net/drill4net-apps.mdx

```diff
@@ -52,9 +51,6 @@ communicatorUrl: 'http://localhost:8090'
 coverageCollectionSettings:
   schedulerDelayInterval: 10 
   collectInitProcessesCoverage: false
-
-supportedTechnologies:
```

Comment:
I though this value is _required_ to enable `WebRequest` instrumentation. Is instrumentation enabled by default?

Reply:
Instrumentation (patching) of HttpClient (since WebRequest is not supported) is enabled or disabled by presence or absence of serviceGroupId config parameter. By default this parameter is absent, so patching is disabled. If user uses microservices it must put value for groupId, the presence of value for groupId  enables patching of HttpClient. For other supported technologies user must call Patch directly in Startup class, the set up process is described in the documentation. On tests project side the patching of HttpClient is enabled by default and there is no way to disable it. Patching for Selenium and Playwright on tests side is enabled manually, the user needs to call the Patch method himself